File: output/llama3.1-8b/avatar/Python/Java/codeforces_612_B.java
import java.util.Scanner;
import java.util.Arrays;

public class codeforces_612_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = Integer.parseInt(scanner.nextLine());
        String[] input = scanner.nextLine().split(" ");
        int[] a = new int[input.length];
        for (int i = 0; i < input.length; i++) {
            a[i] = Integer.parseInt(input[i]);
        }
        int[] b = new int[n];
        for (int i = 0; i < n; i++) {
            b[a[i] - 1] = i;
        }
        int res = 0;
        for (int i = 1; i < n; i++) {
            res += Math.abs(b[i] - b[i - 1]);
        }
        System.out.println(res);
    }
}

File: output/llama3.1-8b/avatar/Java/Python/codeforces_612_B.py
import sys

n = int(input())
dp = [0] * 200001

for _ in range(n):
    idx = int(input())
    dp[idx] = _

ans = 0
for i in range(2, n + 1):
    ans += abs(dp[i] - dp[i-1])

print(ans)